docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -106,6 +106,30 @@ export S3_ENDPOINT_URL='https://<accountid>.r2.cloudflarestorage.com'
 ```
 ````
 
+One can also set credentials through environment variables. For example, with Backblaze:
+
+```{note}
+Your endpoint url is 'https://s3.<your region>.backblazeb2.com'. The account ID can be retrieved through your [Backblaze console](https://secure.backblaze.com/b2_buckets.htm).
+```
+
+````{tabs}
+```{code-tab} py
+import os
+os.environ['S3_ENDPOINT_URL'] = 'https://s3.<your region>.backblazeb2.com'
+os.environ["AWS_ACCESS_KEY_ID"] = '<Backblaze key ID>'
```

Comment:
Let's remove `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` from here and add it in the `Amazon S3` section after the `Your config and credentials files should follow the standard structure output by` line where you can say that one can also set the credentials via environment variables mentioned in [this](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html) link.

Reply:
Done